tasks/lib/string-replace.js

```diff
@@ -94,6 +95,9 @@ exports.init = function(grunt) {
         grunt.log.error(err);
         replace_done(false);
       }
+      if(options.quiet){
+        grunt.log.writeln(chalk.cyan(counter) + ' files created.');
```

Comment:
This might be a nice addition to the logging, but use grunt.log.debug, or grunt.verbose.* to control the output instead of adding unnecessary options 

- http://gruntjs.com/using-the-cli
- http://gruntjs.com/api/grunt.log